packages/elemental-theme/src/custom-elements/ef-time-picker.less

```diff
@@ -55,6 +55,19 @@
       opacity: 1;
     }
   }
+
+  &[error]:not([focused]) {
```

Comment:
Solar theme & Elemental theme doesn't look right on demo page. Let's get them fixed and apply to v7 PR as well.

Reply:
Updated